src/Expenses/expensesControllers.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+const expensesServices = require('./expensesServices');
+
+const getAllExpenses = (req, res) => {
+  res.send(expensesServices.getAll());
+};
+
+const getOneExpense = (req, res) => {
+  const { expenseId } = req.params;
+
+  if (!expenseId) {
+    res.sendStatus(400);
+
+    return;
+  }
```

Comment:
I'm not sure you should do this check because if you don't have valid `expenseId` you won't find `foundExpense` and you  already handled this case further in the code

Reply:
It is tech requirements:
404 if the expected entity doesn't exist (if I have no id)
400 if the required parameter is not passed (if I have an id, but it isn't an appropriate one)